gt/main.c

```diff
@@ -668,6 +830,10 @@ config_gt_instance(struct gt_config *gt_conf, unsigned int lcore_id)
 	int ret;
 	char lua_entry_path[128];
 	unsigned int block_idx = get_block_idx(gt_conf, lcore_id);
+
+	/* Maximum TTL in cycles for each fragmented packet. */
+	uint64_t frag_cycles = (rte_get_tsc_hz() + MS_PER_S - 1) /
+		MS_PER_S * gt_conf->max_flow_ttl_ms;
```

Comment:
True, although we can declare this variable as a const to avoid this variable, I think later we may make all compile-time parameters as run-time parameters, maybe we can keep it as it is here.